cmd/kubeadm/app/cmd/token.go

```diff
@@ -341,7 +341,7 @@ func humanReadableBootstrapToken(token *kubeadmapi.BootstrapToken) string {
 		groupsString = "<none>"
 	}
 
-	return fmt.Sprintf("%s\t%s\t%s\t%s\t%s\t%s\n", token.Token.String(), ttl, expires, usagesString, description, groupsString)
+	return fmt.Sprintf("%s\t%s\t%s\t%s\t%s\t%s", token.Token.String(), ttl, expires, usagesString, description, groupsString)
```

Comment:
@riverzhang are you basically removing an extra empty line here because `fmt.Fprintln(w, humanFriendlyTokenOutput)` already adds one?

Reply:
never mind i missed the linked issue:
https://github.com/kubernetes/kubeadm/issues/1094

thanks
/lgtm